test/event_test.rb

```diff
@@ -63,6 +64,15 @@ def test_categories_are_objects
     assert_kind_of Chiketto::Category, @event.category
     assert_kind_of Chiketto::Category, @event.subcategory
   end
+
+  def test_questions_are_objects
+    find_event
+    VCR.use_cassette 'event-questions' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.